### PR TITLE
Use default cosign-installer version

### PR DESCRIPTION
**Pull request by hectorj2f**
#### Summary

Stop setting the cosign version in our gh actions, and use the default cosign version from the installer.

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -36,8 +36,6 @@ jobs:
       - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
 
       - uses: sigstore/cosign-installer@c3667d99424e7e6047999fb6246c0da843953c65
```
Comment (cpanato) [inline .github/workflows/build.yaml:38]: can you also check if this hash is from the latest release?

Comment (hectorj2f) [reply .github/workflows/build.yaml:38]: yes it is

Comment (hectorj2f) [reply .github/workflows/build.yaml:38]: oh, it got merged a new PR 23min ago.

Comment (cpanato) [reply .github/workflows/build.yaml:38]: thanks

**Pull request merged**